06-Job-Declaration-Protocol.md

```diff
@@ -38,7 +38,7 @@ Notably, if the pool intends to change the space it requires for coinbase transa
 | ----------------------------------- | --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | request_id                          | U32       | Unique identifier for pairing the response                                                                                                                                                                                                                                                                                                                               |
 | mining_job_token                    | B0_255    | Token that makes the client eligible for committing a mining job for approval/transaction declaration or for identifying custom mining job on mining connection.                                                                                                                                                                                                         |
-| coinbase_output_max_additional_size | U32       | The maximum additional serialized bytes which the pool will add in coinbase transaction outputs. See discussion in the Template Distribution Protocol's CoinbaseOutputDataSize message for more details.                                                                                                                                                                 |
+| coinbase_output_max_additional_size | U32       | The maximum additional serialized bytes which the pool will add in coinbase transaction outputs. See discussion in the Template Distribution Protocol's CoinbaseOutputConstraints message for more details.                                                                                                                                                                 |
```

Comment:
as pointed out here https://github.com/stratum-mining/sv2-spec/pull/86#issuecomment-2447119433 

we need to add `coinbase_output_max_sigops` to `AllocateMiningJobToken.Success`